codequality/configs/codestyles/CRiSolutionsAndroid.xml

```diff
@@ -282,15 +481,6 @@
     </arrangement>
   </codeStyleSettings>
   <codeStyleSettings language="kotlin">
-    <option name="CODE_STYLE_DEFAULTS" value="KOTLIN_OFFICIAL" />
-    <option name="CALL_PARAMETERS_WRAP" value="0" />
-    <option name="CALL_PARAMETERS_LPAREN_ON_NEXT_LINE" value="false" />
-    <option name="CALL_PARAMETERS_RPAREN_ON_NEXT_LINE" value="false" />
-    <option name="METHOD_PARAMETERS_WRAP" value="0" />
-    <option name="METHOD_PARAMETERS_LPAREN_ON_NEXT_LINE" value="false" />
-    <option name="METHOD_PARAMETERS_RPAREN_ON_NEXT_LINE" value="false" />
-    <option name="EXTENDS_LIST_WRAP" value="0" />
-    <option name="METHOD_CALL_CHAIN_WRAP" value="0" />
-    <option name="ASSIGNMENT_WRAP" value="0" />
+    <option name="FIELD_ANNOTATION_WRAP" value="1" />
```

Comment:
Looks like I have different values for this for Java and Kotlin. We might want to make these consistent.

Reply:
Pulled the repo and checked the diffy agreed it seems like this is the only difference, and maybe some of the older code was in a different formatting.
That being said, in Kotlin we would not wrap fields but looks like in Java we are. For consistency sake I'd say we merge this in. What are your thoughts?

Reply:
I updated Java to wrap after 1 annotation per line to match Kotlin.